packages/nouns-contracts/contracts/StreamEscrow.sol

```diff
@@ -0,0 +1,307 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/// @title Stream Escrow
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.19;
+
+import { IStreamEscrow } from './interfaces/IStreamEscrow.sol';
+import { INounsToken } from './interfaces/INounsToken.sol';
+
+contract StreamEscrow is IStreamEscrow {
+    event ETHStreamedToDAO(uint256 amount);
```

Comment:
in most of our code I think we have events in interfaces, wondering if we should do the same here for consistency?

Reply:
sure a150478703b769dd24716c0077986ae170ebf23e